packages/launchpad/src/components/ButtonBar.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import ButtonBar from './ButtonBar.vue'
+
+describe('<ButtonBar />', () => {
+  it('playground', () => {
+    cy.mount(() => <ButtonBar next="Next Step" back="Back" />)
```

Comment:
Happy to use TSX here, kind of surprised it's not messing with React TSX (in terms of types).

Any benefit to TSX over 

```ts
mount(ButtonBar, {
    props: { next: 'Next Step' }
  }
})
```

?

Reply:
Easier to read ;)